packages/eslint-plugin/src/utils/package-json-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { ProjectFileMap, readJsonFile } from '@nx/devkit';
+
+export function getAllDependencies(path: string) {
```

Comment:
A return type would be nice, especially for exported functions `Record<string, string>`.